web/src/DetailedInfoPage.tsx

```diff
@@ -427,17 +424,14 @@ export const DetailedInfoPage: React.FunctionComponent<{}> = () => {
                     .filter(([key, value]) =>
                         sampleFieldsToDisplay.includes(key)
                     )
-                    .map(([key, value]) =>
-                        value ? (
-                            <div key={`${key}-${value}`}>
-                                <b>{key}:</b> {value.toString()}
-                            </div>
-                        ) : (
-                            <React.Fragment
-                                key={`${key}-${value}`}
-                            ></React.Fragment>
-                        )
-                    )}
+                    .map(([key, value]) => (
+                        <div key={`${key}-${value}`}>
+                            <b>{key}:</b>{" "}
+                            {value != null
+                                ? value.toString()
+                                : "null/undefined"}
```

Comment:
Could use typescript optional coalescing to simplify this, and I think other people might find `null` confusing.

```suggestion
                            {value?.toString() ?? <em>no value</em>}
```

Reply:
Oooh yes smart, will fix this after we discuss the other thing